NitroxPatcher/Patches/Dynamic/InfectionReveal_Patch.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using HarmonyLib;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class InfectionReveal_Patch : NitroxPatch, IDynamicPatch
+{
+    internal static readonly MethodInfo TARGET_METHOD = AccessTools.EnumeratorMoveNext(Reflect.Method((Player t) => t.TriggerInfectionRevealAsync()));
+    
+    public static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions)
+    {
+        /*
+From:
+            if (!this.infectionRevealed){
+    float num = this.armsController.StartHolsterTime(12f);
+
+To:
+            if (!this.infectionRevealed){
+    float num = this.armsController.StartHolsterTime(12f);
+    SendInfectAnimationStartPacket();
+ */
+        return new CodeMatcher(instructions)
+       .MatchStartForward(
+           new CodeMatch(OpCodes.Ldloc_1),
+           new CodeMatch(OpCodes.Ldfld),
+           new CodeMatch(OpCodes.Ldc_R4),
+           new CodeMatch(OpCodes.Callvirt, Reflect.Method((ArmsController arms) => arms.StartHolsterTime(default)))
+       )
+       .Insert(
+           new CodeInstruction(OpCodes.Call, Reflect.Method(() => SendInfectAnimationStartPacket()))
+       )
+       .InstructionEnumeration();
```

Comment:
Minor thing but in the codebase in multi call lines the first call stays in the original line  
```suggestion
        return new CodeMatcher(instructions).MatchStartForward(
                                                new CodeMatch(OpCodes.Ldloc_1),
                                                new CodeMatch(OpCodes.Ldfld),
                                                new CodeMatch(OpCodes.Ldc_R4),
                                                new CodeMatch(OpCodes.Callvirt, Reflect.Method((ArmsController arms) => arms.StartHolsterTime(default)))
                                            )
                                            .Insert(
                                                new CodeInstruction(OpCodes.Call, Reflect.Method(() => SendInfectAnimationStartPacket()))
                                            )
                                            .InstructionEnumeration();
```